src/modelAnalysis/FVA/fluxVariability.m

```diff
@@ -167,25 +180,32 @@
      PCT_status=0;  % Parallel Computing Toolbox not found.
 end
 
-
-
-if PCT_status &&(~exist('parpool') || poolsize == 0)  %aka nothing is active
+if ~PCT_status &&(~exist('parpool') || poolsize == 0)  %aka nothing is active

```

Comment:
Wouldn't it be sufficient if PCT_status is 0 ?
In addition: poolsize needs to be initialized further up (i.e. in the PCT check else statement poolsize needs to be set to 0), or this throws an error if "parpool" exists (for whatever reason), could be a user function...